src/client/src/orangehrmClaimPlugin/components/ClaimAttachment.vue

```diff
@@ -0,0 +1,271 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <oxd-divider class="orangehrm-horizontal-margin" />
+  <div class="orangehrm-horizontal-padding orangehrm-vertical-padding">
+    <div class="orangehrm-action-header">
+      <oxd-text tag="h6" class="orangehrm-main-title">
+        {{ $t('general.attachments') }}
+      </oxd-text>
+      <oxd-button
+        :label="$t('general.add')"
+        icon-name="plus"
+        display-type="text"
+        @click="onClickAdd"
+      />
+    </div>
+  </div>
+  <table-header
+    :total="total"
+    :loading="isLoading"
+    :selected="checkedItems.length"
+    @delete="onClickDeleteSelected"
+  />
+  <div class="orangehrm-container">
+    <oxd-card-table
+      v-model:selected="checkedItems"
+      :items="items.data"
+      :headers="tableHeaders"
+      :selectable="canEdit"
+      :clickable="false"
+      :loading="isLoading"
+      row-decorator="oxd-table-decorator-card"
+    />
+  </div>
+  <delete-confirmation ref="deleteDialog"></delete-confirmation>
+</template>
+
+<script>
+import {APIService} from '@/core/util/services/api.service';
+import usePaginate from '@ohrm/core/util/composable/usePaginate';
+import DeleteConfirmationDialog from '@ohrm/components/dialogs/DeleteConfirmationDialog.vue';
+import {convertFilesizeToString} from '@ohrm/core/util/helper/filesize';
+
+export default {
+  name: 'ClaimAttachment',
+
+  components: {
+    'delete-confirmation': DeleteConfirmationDialog,
+  },
+  props: {
+    requestId: {
+      type: Number,
+      required: true,
+    },
+    canEdit: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  setup(props) {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      `/api/v2/claim/requests/${props.requestId}/attachments`,
+    );
+
+    const attachmentDataNormalizer = (data) => {
+      return data.map((item) => {
+        return {
+          id: item.id,
+          attachedDate: item.date ? item.date : '',
+          filename: item.attachment.fileName ? item.attachment.fileName : '',
```

Comment:
```suggestion
          filename: item.attachment.fileName ?? "",
```
[Nullish collacesing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)